test/benchmark_records.md

```diff
@@ -76,3 +76,72 @@ macros to guarantee that their hashes are computed at compile time.
   11 │ dataframes  sha256     543.125 μs  749.125 μs   1.37929
   12 │ numbers     sha256     270.958 μs  371.833 μs   1.37229
 ```
+
```

Comment:
I haven't updated any of these parts for the current PR; this benchmark includes the caching.